_overviews/scala3-book/control-structures.md

```diff
@@ -144,26 +144,14 @@ scala> for i <- ints do println(i)
 3
 ````
 
-When you need a multiline block of code following the `for` generator, use either of these approaches:
+When you need a multiline block of code following the `for` generator, use the following syntax:
 
 ```scala
-// option 1
 for
   i <- ints
 do
   val x = i * 2
   println(s"i = $i, x = $x")
-
-// option 2
-for (i <- ints)
-  val x = i * 2
-  println(s"i = $i, x = $x")
-
-// option 3
-for (i <- ints) {
-  val x = i * 2
-  println(s"i = $i, x = $x")
-}
```

Comment:
I suggest not showing too many options, unless there is a clear rule that says which one to pick when. Here, I would just show the preferred option.

Reply:
Sounds good to me!